mivs/templates/mivs_judging/index.html

```diff
@@ -16,7 +16,7 @@
 <tbody>
 {% for game in games %}
     <tr>
```

Comment:
I figured this link would make it easier for admins and judges to send links to to the studios.  Though we might end up wanting to restrict this to only admins and not judges?  Seems a little paranoid but I'll have to think about it.